staging/src/k8s.io/apiserver/pkg/endpoints/request/webhook_duration.go

```diff
@@ -96,6 +116,37 @@ type LatencyTrackers struct {
 	// ValidatingWebhookTracker tracks the latency incurred in validating webhook(s).
 	// Validate webhooks are done in parallel, so max function is used.
 	ValidatingWebhookTracker DurationTracker
+
+	// StorageTracker tracks the latency incurred inside the storage layer,
+	// it accounts for the time it takes to send data to the underlying
+	// storage layer (etcd) and get the complete response back.
+	// If a request involves N (N>=1) round trips to the underlying
+	// stogare layer, the latency will account for the total duration
+	// from these N round trips.
+	// It does not include the time incurred in admission, or validation.
+	StorageTracker DurationTracker
+
+	// TransformTracker tracks the latency incurred in transforming the
```

Comment:
transforming from what to what.

Reply:
This includes transforming the object to user's desired form (ie. as Table), and also setting appropriate API level fields. 
If you think we don't need it now that we are tracking serialization and response write separately, i can remove it.

Reply:
> This includes transforming the object to user's desired form (ie. as Table), and also setting appropriate API level fields.

I just want the comment to explain that.  "transform" is pretty vague.